toolshed/build_static_bitcode_input.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+"""
+Helper to produce static bitcode input for test_nvvm.py.
+
+Usage:
+    python toolshed/build_static_bitcode_input.py
+
+It will print a ready-to-paste MINIMAL_NVVMIR_BITCODE_STATIC entry for the
+current NVVM IR version detected at runtime.
+"""
+
+import binascii
+import os
+import sys
+import textwrap
+
+import llvmlite.binding  # HINT: pip install llvmlite
+from cuda.bindings import nvvm
+
+
+def get_minimal_nvvmir_txt_template():
```

Comment:
Can't you just check in a JSON file that contains the bitcode? Why do we need the `sys.path` munging and `print`-generated dictionary?